src/Form/SettingsForm.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Drupal\os2forms_dig_sig_server\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+
+/**
+ * Signing settings form.
+ */
+class SettingsForm extends ConfigFormBase {
+  use StringTranslationTrait;
+
+  /**
+   * Name of the config.
+   *
+   * @var string
+   */
+  public static $configName = 'os2forms_dig_sig_server.settings';
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'os2forms_dig_sig_server_settings';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return [self::$configName];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $form['signing_service_url'] = [
+      '#type' => 'textfield',
```

Comment:
```suggestion
      '#type' => 'url',
```

cf. https://www.drupal.org/docs/drupal-apis/form-api/form-render-elements#s-html5-form-render-elements. I'm not sure if it works.